er_aws_kms/__main__.py

```diff
@@ -0,0 +1,58 @@
+import os
+from pathlib import Path
+
+from external_resources_io.input import parse_model, read_input_from_file
+
+from er_aws_kms.input import AppInterfaceInput
+
+
+def get_ai_input() -> AppInterfaceInput:
+    """Get the AppInterfaceInput from the input file."""
+    return parse_model(
+        AppInterfaceInput,
+        read_input_from_file(
+            file_path=os.environ.get("ER_INPUT_FILE", "/inputs/input.json"),
+        ),
+    )
+
+
+def create_tf_vars_json(
+    app_interface_input: AppInterfaceInput, vars_file: str = "./module/tfvars.json"
+) -> None:
+    """Create the terraform vars file"""
+    path = Path(vars_file)
+    with path.open(mode="w", encoding="utf-8") as file:
+        file.write(app_interface_input.data.model_dump_json(exclude_none=True))
+
+
+def create_backend_tf_file(
+    app_interface_input: AppInterfaceInput, backend_file: str = "./module/backend.tf"
```

Comment:
`backend_file` should be configurable via environment variable

Reply:
why?

Reply:
see above